src/icon/fill/FileCode16.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+/* eslint-disable react/jsx-sort-props */
+import * as React from 'react';
+import AccessibleSVG from '../../icons/accessible-svg';
+import type { Icon } from '../../icons/flowTypes';
+
+const FileCode16 = (props: Icon) => (
+    <AccessibleSVG width={16} height={16} viewBox="0 0 16 16" {...props}>
```

Comment:
Are we intentionally removing the classNames? Like in this case it would have been `icon-file-code` previously?

Reply:
I have a follow up step to add in class names per component, among some other changes (to handle color variables, and better documentation generation).